src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="container h-75">
+  <div class="row h-100 justify-content-center align-items-center">
+    <div class="blank-page">
+      <h2 i18n>Please create your first cluster</h2>
+
+      <div class="offset-md-3">
+        <button class="btn btn-accent m-2"
+                [routerLink]="'/dashboard'"
+                (click)="createCluster()"
+                i18n>Create Cluster</button>
+        <button class="btn btn-light"
+                [routerLink]="'/dashboard'"
+                i18n>Skip</button>
```

Comment:
Aren't we setting the status=post_installed here?

Reply:
Missed it but done now